designs/administrator/future-openshift/cluster-settings/index.md

```diff
@@ -0,0 +1,64 @@
+---
+parent: Adminstrator
+version: 4.6
+---
+
+# Cluster Settings 
+
+Users complete upgrades in cluster settings. The following design focuses on making improvements to the current upgrade experience in cluster settings by making upgrading as simple and transparent as possible. Specific improvements include adding more information around recommended upgrade paths and notifying users when new versions and channels become available. 
+
+## Adding Alerts to the Notification Drawer
```

Comment:
```suggestion
## Adding Recommendation Alerts to the Notification Drawer
```

Reply:
As soon as I get confirmation I will make this change.

Reply:
Icon looks good now!